the-collab-lab/tcl-26-smart-shopping-list
# Connect app to firebase (#1)

## Description

We installed firebase and react-firebase-hooks as dependencies. We added two sample components, TestDbWrite and TestDbRead, to test reading and writing to the Firestore database. TestDbWrite contains a button that adds sample data. TestDbRead prints a list of stringified entries.

|     | Type                       |
| --- | -------------------------- |
|  ✓    | :sparkles: New feature     |
|  ✓    | :link: Update dependencies |

## Acceptance Criteria


## Related Issue

Closes #1

## Testing Steps / QA Criteria

- Run npm install to install firebase and react-firebase-hooks.
- Run npm start to start the app on your localhost.
- Click the "Add to Firestore" button to add some test data, and you should see an extra item appear in the list each time you click the button. [Check the database for changes here](https://console.firebase.google.com/project/tcl-26-shopping-list/firestore/data/).
